students/Lina/Lesson3/mailroom.py

```diff
@@ -0,0 +1,146 @@
+#! python
+
+#------------------------------------------------
+# Lesson 3 - Assignment 1: Mailroom, Part 1
+#------------------------------------------------
+
+import sys
+from operator import itemgetter
+from datetime import date
+
+#This list contains the donor names and a history of their donation amounts.
+donor_db = [("William Gates, III", [653772.32, 12.17]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [663.23, 43.87, 1.32]),
+            ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),
+            ("Oprah Winfrey", [3312.00, 5500.00]),
+            ("Joe Smith", [250.75, 120.00, 55.50, 100.50])
+            ]
+
+def show_donor_list():
+    #This function will show all donors in the list.
+    for row in donor_db:
+        print(row[0])
+
+def verify_donor(name):
+    #This function will check if the donor is in the list. If not there,
+    #it will add to the list.
+    new_donor = True
+    for row in donor_db:
+        if row[0] == name:
+            new_donor = False
+            break
+
+    if new_donor:
```

Comment:
instead of using a variable you can use an else clause with for loop:
```
for donor in donor_db:
    if donor[0] == name:
        # update
        break
else:
    # goes here if not found
```